### ISSUE TITLE
Bun ignores `--dotenv` option

### ISSUE THREAD

**Issue by ahas** [nuxt/cli#301]
### Environment

- Operating System: Linux
- Node Version:     v18.17.1
- Nuxt Version:     3.8.2
- CLI Version:      3.10.0
- Nitro Version:    2.8.0
- Package Manager:  bun@1.0.14
- Builder:          -
- User Config:      devtools, app
- Runtime Modules:  -
- Build Modules:    -

### Reproduction

1. Create multiple `.env` files: `.env.local`, `.env.dev`, `.env.prod`.
2. Add `--dotenv .env.dev` to the dev command in `package.json` > `scripts`.
For example: ```"dev": "nuxt dev --dotenv .env.dev"```
Even if executed with ```bunx nuxt dev --dotenv .env.dev```, the result is the same.
3. Write code that utilizes values from the `.env.dev` file, execute the `dev` command, and check the result.

### Describe the bug

When running Nuxt with Bun, the `--dotenv` option does not work as specified.
If both `.env.local` and `.env.dev` files are created, it seems that the `.env.local` file is read with priority.

**Comment by OrbisK** [comment]
@ahas are you still facing this issue with bun? is there an upstream issue open at bun?

**Comment by ahas** [comment]
> [@ahas](https://github.com/ahas) are you still facing this issue with bun? is there an open upstream issue at bun?

No. At this point, the bug no longer occurs. I will close the issue.
